src/SourceBrowser.Generator/DocumentWalkers/DocumentWalker.cs

```diff
@@ -180,6 +181,18 @@ private string GetSymbolName(ISymbol symbol)
                 var containingName = symbol.ContainingSymbol.ToString();
                 fullyQualifiedName = containingName + _walkerUtils.LocalVariableDelimiter + symbol.Name;
             }
+            else if(symbol.Kind == SymbolKind.Method)
+            {
+                var methodSymbol = (IMethodSymbol)symbol;
+                if (methodSymbol.ReducedFrom != null)
+                {
+                    fullyQualifiedName = methodSymbol.ReducedFrom.ToString();
```

Comment:
Linking to extension methods is a difficult topic, and I think it warrants a comment here